Cannot find module 'src/i18n/i18n.service'

<!-- Feel free to remove sections that aren't relevant.

## Title line template: [Title]: Compilation error

-->

## Detailed description

> Describe in detail the issue you're having.

After updating `carbon-components-angular` to version `3.14.5`, the following compilation errors for many `d.ts` files occur:
error TS2307: Cannot find module 'src/i18n/i18n.service'.

> Is this a feature request (new component, new icon), a bug, or a general issue?

Bug

> What version of the Carbon Design System are you using?

carbon-components: 10.5.1
carbon-components-angular: 3.14.5

## Additional information

* Screenshots or code
![image](https://user-images.githubusercontent.com/43789093/63336145-548afd00-c347-11e9-95e4-e1208b9a494d.png)

* Notes
There are no such errors with version `3.14.4`

--- a/src/i18n/i18n.module.ts
+++ b/src/i18n/i18n.module.ts
@@ -3,15 +3,15 @@ import { NgModule, SkipSelf, Optional } from "@angular/core";
 import { I18n } from "./i18n.service";
 import { ReplacePipe } from "./replace.pipe";
 
-export { I18n, replace } from "./i18n.service";
+export { I18n, replace, Overridable } from "./i18n.service";
 export { ReplacePipe } from "./replace.pipe";
 
-// either provides a new instance of ModalPlaceholderService, or returns the parent
+// either provides a new instance of I18n, or returns the parent
 export function I18N_SERVICE_PROVIDER_FACTORY(parentService: I18n) {
 	return parentService || new I18n();
 }
 
-// placholder service *must* be a singleton to ensure the placeholder viewref is accessible globally
+// I18n should provide a single instance of itself to ensure that translations are consistent through the app
 export const I18N_SERVICE_PROVIDER = {
 	provide: I18n,
 	deps: [[new Optional(), new SkipSelf(), I18n]],

--- a/src/pagination/pagination.component.ts
+++ b/src/pagination/pagination.component.ts
@@ -6,7 +6,7 @@ import {
 	EventEmitter
 } from "@angular/core";
 
-import { I18n } from "./../i18n/i18n.module";
+import { I18n, Overridable } from "./../i18n/i18n.module";
 import { ExperimentalService } from "./../experimental.module";
 import { merge } from "./../utils/object";
 

--- a/src/table/table.component.ts
+++ b/src/table/table.component.ts
@@ -15,7 +15,7 @@ import { TableHeaderItem } from "./table-header-item.class";
 import { TableItem } from "./table-item.class";
 import { getScrollbarWidth } from "../common/utils";
 import { getFocusElementList, tabbableSelectorIgnoreTabIndex } from "../common/tab.service";
-import { I18n } from "./../i18n/i18n.module";
+import { I18n, Overridable } from "./../i18n/i18n.module";
 import { merge } from "./../utils/object";
 
 export interface TableTranslations {

--- a/src/table/toolbar/table-toolbar.component.ts
+++ b/src/table/toolbar/table-toolbar.component.ts
@@ -1,6 +1,6 @@
 import { TableModel } from "../table-model.class";
 import { Component, Input } from "@angular/core";
-import { I18n } from "../../i18n/i18n.module";
+import { I18n, Overridable } from "../../i18n/i18n.module";
 
 /**
  * The table toolbar is reserved for global table actions such as table settings, complex filter, export, or editing table data.

--- a/src/tiles/expandable-tile.component.ts
+++ b/src/tiles/expandable-tile.component.ts
@@ -4,7 +4,7 @@ import {
 	ElementRef,
 	AfterContentInit
 } from "@angular/core";
-import { I18n } from "./../i18n/i18n.module";
+import { I18n, Overridable } from "./../i18n/i18n.module";
 import { merge } from "./../utils/object";
 
 export interface ExpandableTileTranslations {

--- a/src/toggle/toggle.component.ts
+++ b/src/toggle/toggle.component.ts
@@ -9,7 +9,7 @@ import {
 } from "@angular/core";
 import { NG_VALUE_ACCESSOR } from "@angular/forms";
 
-import { I18n } from "../i18n/i18n.module";
+import { I18n, Overridable } from "../i18n/i18n.module";
 import { Observable } from "rxjs";
 
 /**

